datumaro/plugins/icdar_format/converter.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import os.path as osp
+
+from datumaro.components.converter import Converter
+from datumaro.components.extractor import AnnotationType, CompiledMask
+from datumaro.util.image import save_image
+from datumaro.util.mask_tools import find_mask_bbox, paint_mask
+
+from .format import IcdarPath, IcdarTask
+
+
+class _WordRecognitionConverter():
+    def __init__(self):
+        self.annotations = ''
+
+    def save_categories(self, save_dir, categories):
+        pass
+
+    def save_annotations(self, item, categories):
+        self.annotations += '%s, ' % (item.id + IcdarPath.IMAGE_EXT)
+        for ann in item.annotations:
+            if ann.type != AnnotationType.caption:
+                continue
+            self.annotations += '\"%s\"' % ann.caption
+        self.annotations += '\n'
+
+    def write(self, path):
+        file = osp.join(path, 'gt.txt')
+        os.makedirs(osp.dirname(file), exist_ok=True)
+        with open(file, 'w') as f:
+            f.write(self.annotations)
+
+    def is_empty(self):
+        return len(self.annotations) == 0
+
+class _TextLocalizationConverter():
+    def __init__(self):
+        self.annotations = {}
+
+    def save_categories(self, save_dir, categories):
+        vocabulary_file = osp.join(save_dir,
```

Comment:
I still don't known if is a good idea or not to store text in the `label` field. It might be better to introduce `CaptionCategories` with vocabulary, if it is provided, or just use an attribute for text.